solution/ui/regulations/eregs-vite/src/components/header/HeaderLinks.vue

```diff
@@ -0,0 +1,84 @@
+<script setup>
+import { ref } from "vue";
+
+import HeaderChevronUp from "../svgs/header-chevron-up.vue";
+import HeaderChevronDown from "../svgs/header-chevron-down.vue";
```

Comment:
as mentioned above, relative pathing is needed for any components that are used in `eregs-component-lib` -- ironing out the aliases in the config for both Vite pipelines is TODO